docs/book/src/getting-started.md

```diff
@@ -47,7 +47,7 @@ it into a management cluster using `clusterctl`.
    > `${ServiceRegion1}:${ServiceID1}=${URL1},${ServiceID2}=${URL2};${ServiceRegion2}:${ServiceID1}=${URL1...}`.
    
 
-    Supported ServiceIDs include - `vpc, powervs, rc`
+    Supported ServiceIDs include - `CIS, COS, DNSServices, IAM, KeyProtect, ResourceManager, powervs, rc, vpc`
```

Comment:
As of today, We dont user any of CIS, DNSServices, KeyProtect services. so should we add them now or add them when we start using these services later.

Reply:
make sense, lets remove the entries which aren't used in this controller at the moment.

Reply:
@Karthik-K-N is thee ResourceManager duplicate of rc here?

Reply:
@Karthik-K-N also the `iam` endpoint is being set via `IBMCLOUD_AUTH_URL` param, so do we really need the IAM service ID?

Reply:
I think so its duplicate, if we want to use ResourceManager instead of rc, may be we can add support for both for a release and based on requirement we can deprecate later.

Reply:
IMO its a duplicate and not required - https://pkg.go.dev/github.com/IBM/platform-services-go-sdk/resourcemanagerv2#pkg-constants

Reply:
Regarding iam, I think there is no way to use anything other than IBMCLOUD_AUTH_URL right, as its internally used by client?

Reply:
Yes

> Regarding iam, I think there is no way to use anything other than IBMCLOUD_AUTH_URL right, as its internally used by client?

Yes, so having `iam` service id doesn't not make sense here i feel

Reply:
additional information... we have our set of required endpoint services for OCP defined here: https://github.com/openshift/installer/blob/release-4.15/data/data/install.openshift.io_installconfigs.yaml#L3450-L3492

we default to public endpoints for these required services, but provide the ability to override via install-config arguments

that served our basis for this PR

Reply:
Thanks for the information, not all those service endpoints from openshift will be used by the CAPI controller, e.g: CIS will be used by the ingress operator(part of OCP Cluster Operartors) which is out of scope for this project.

We will add which are really required for the CAPI controllers to run e.g: COS, IAM, ResourceController VPC etc..

Reply:
I can add them later, but they could all possibly be required (depending on the Network design) as the Infrastructure resources get created/referenced in VPC related enhancements.

Or we can wait on merging this until I have completed the necessary enhancements to CAPI for VPC Infrastructure, if that appears to be a better option.

Reply:
Let's have an incremental approach, you can merge the code with bare minimum required changes, later will add those additional endpoints as in when needed.